### ISSUE TITLE
UX Designs

### ISSUE THREAD

**Issue by celsom3** [Cosecha/redadalertas#23]
We could use some help from designers to make our stuff look nice. 

It would be cool to use [Material Design](https://en.wikipedia.org/wiki/Material_Design) or something like [Semantic UI](http://react.semantic-ui.com/). Mainly because there are react component libraries for both.

Just a matter of choosing one and going with it. But there needs to be more discussion around this.

**Comment by celsom3** [comment]
I personally like Material, but it would mean ejecting out of CRA. And I'm scared lol. Mainly because I've tried it once and didn't know how to set up SASS in webpack. But if someone can do it, I'm in.

**Comment by celsom3** [comment]
The Material React library I'm thinking of is http://react-toolbox.com/#/

**Comment by pfarnach** [comment]
I've never used CRA before, just "manually" configured webpack, so I should be able to handle that stuff.  I'm not going to have any time until later this week or next weekend so if someone else wants to tackle it first, go ahead.

I haven't used it before, but React-Toolbox seems like a solid choice. 

A side note -- looking at versions, unfortunately both React Toolbox and Material UI have new major versions right around the corner.  This means we'd be developing with soon-to-be outdated versions unless we want to roll the dice with React Toolbox's beta.

**Comment by celsom3** [comment]
Yeah, I've noticed that. I would probably roll my dice with react-toolbox. Seems like either way, there will be breaking changes.

**Comment by 8bakon8** [comment]
Material and semantic look very similar... i'm not very experienced with using either but i've seen examples. Material is very minimal in it's design, I suggest using whatever you're comfortable with. I can start working on a template tonight. 

The other designer in my team is alot more experienced when it comes to web. she'll be joining us soon

**Comment by zer0kool** [comment]
http://materializecss.com/

**Comment by chrysochlorous** [comment]
What is the current status of the UX development? I'd like to help out if there is still a need. I have experience with React, Vue, CodeKit, npm, SASS, and more.

**Comment by arghgr** [comment]
@chrysochlorous Hey, we actually need UX people for the mobile app. This particular repo is not currently in use (this repo is more for making reports, and we're focusing on the alerts at the moment) and our conversations are happening on Slack, so please send me an email (b@argh.gr) and I'll get you on there.
